src/h3lib/include/h3Index.h

```diff
@@ -179,4 +180,8 @@ H3Index _h3Rotate60ccw(H3Index h);
 H3Index _h3Rotate60cw(H3Index h);
 DECLSPEC H3Index _zeroIndexDigits(H3Index h, int start, int end);
 
+H3Error vec3dToCell(const Vec3d *v, int res, H3Index *out);
```

Comment:
```suggestion
H3Error vec3ToCell(const Vec3d *v, int res, H3Index *out);
```
Some of the function names are inconsistent about whether it's a vec3 or vec3d etc, at least in new and modified functions we can be consistent on vec3?